package.json

```diff
@@ -43,6 +44,7 @@
     "lodash.pick": "^4.4.0",
     "napi-macros": "^2.0.0",
     "node-fetch": "^2.6.0",
+    "node-gyp-build": "^4.1.0",
```

Comment:
Shouldn't this be in `devDependencies`?

Reply:
@types/node probably yes, but node-gyp-build is always needed, it loads the prebuilds and wraps node-gyp. See binding.js

Reply:
If I understand it correctly, `dependencies` should only be those required at run-time. And then `devDependencies` certainly for build-time dependencies. Dependencies required for testing, linting, etc. can go to `devDependencies` or `optionalDependencies`.
But that's only what I gathered from reading about it for 20min a couple of days ago.

Reply:
Yes, but node-gyp-build is required at runtime, because it loads the prebuild/native lib